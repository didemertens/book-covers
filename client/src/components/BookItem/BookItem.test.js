// import React from 'react'
// import TestRenderer from 'react-test-renderer'
// import BookItem from './BookItem'


// describe('The list of book covers', () => {
//   it('renders the list', () => {
//     const tree = TestRenderer.create(
//       <BookItem bookData={
//         [
//           {
//             "id": 1,
//             "title": "Pond",
//             "author": "Claire-Louise Bennett",
//             "cover": "/covers/pond-bennett.jpg"
//           },
//           {
//             "id": 2,
//             "title": "Late Migrations: A Natural History of Love and Loss",
//             "author": "Margaret Renkl",
//             "cover": "/covers/migrations-renkl.jpg"
//           }
//         ]
//       } />
//     )
//     console.log(tree.toJSON())

//     expect(tree.toJSON()).toMatchSnapshot()
//   })
// })