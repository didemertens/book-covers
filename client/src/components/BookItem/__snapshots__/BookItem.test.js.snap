// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The list of book covers renders the list 1`] = `
Array [
  <div>
    <img
      alt="Pond"
      src="/covers/pond-bennett.jpg"
    />
  </div>,
  <div>
    <img
      alt="Late Migrations: A Natural History of Love and Loss"
      src="/covers/migrations-renkl.jpg"
    />
  </div>,
]
`;
